examples/animation.js

```diff
@@ -44,10 +44,11 @@ app.module.directive('appMap', app.mapDirective);
 
 /**
  * The application's main controller.
+ * @param {angular.$timeout} $timeout Angular timeout service.
```

Comment:
It's not actually required for the ngeo examples but we should use `@ngInject` where appropriate to show good practices.